### DISCUSSION TITLE
Tekmar 486 Gateway

### DISCUSSION THREAD

**Discussion by Pedalmaniac** [WillCodeForCats/tekmar-482#89] [General]
Are there any future plans for including the 486 gateway in the Tekmar HACs?

**Comment by WillCodeForCats** [comment]
No, because the 486 does not have an API to connect to. The 486 is cloud-only.

If Tekmar ever decides to update it in the future I will include it.

**Reply by Pedalmaniac** [reply]
Thanks. I realised that after reading some more of the wiki. Oh well.

**Reply by WillCodeForCats** [reply]
I have a 486 and I've port scanned it to be sure there weren't any ports open to access it. I use my 482 as permanently connected for HA and only bring the 486 out for testing in case they ever release an update for it.
